src/Mapper.js

```diff
@@ -480,8 +480,18 @@ function Mapper (opts) {
         return subClass
       })()
     })
+  } else if (this.recordClass !== false) {
+    const customSuperClass = this.recordClass
+    this.recordClass = function newRecordClass (props, opts) {
+      var base = Reflect.construct(customSuperClass, arguments, newRecordClass); // eslint-disable-line
```

Comment:
How does `Reflect.construct` transpile to ES5? I've never used it before.

Reply:
I reviewed our webpack output, and it appears it doesn't appear to convert it to es5. 

I am not familiar enough with this part of js to know a path forward with ES5.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/construct